### DISCUSSION TITLE
starship.rs

### DISCUSSION THREAD

**Discussion by JoshPaulie** [catppuccin/catppuccin#1927] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port
- [X] [Done](https://github.com/JoshPaulie/starship)! 🚀

### How can the requested port be themed?

[starship.rs](https://starship.rs) is a customizable prompt. one can define custom palettes and use defined color names, perfect for Catppuccin.

### Has the requested port been themed already?

Yes & no. Others have proposed ports, but were static opinionated. With the addition of user defined palettes, a much more flexible "port" is now possible.

**Comment by sgoudham** [comment]
Hey :wave: 

As discussed on discord, we, the staff team, are avoiding our previous stance (https://github.com/catppuccin/catppuccin/issues/203#issuecomment-1202140218) on the inclusion of shell prompts for this port request. 

Previous attempts at inclusion were denied due to the opinionated nature of those attempts but this request simply provides a defined Catppuccin palette and therefore, we believe it is a valid port request :+1:

**Reply by sgoudham** [reply]
Transferred to #1929
